### DISCUSSION TITLE
[HxMessenger] and [HxToast] Usage

### DISCUSSION THREAD

**Discussion by econner20** [havit/Havit.Blazor#777] [Q&A] [answered]
In the documentation it states that HxToast is not to be used in the user code but rather use HxMessenger. I see methods for AddInformation, AddWarning and AddError. However, how can the HxMessenger be configured like the HxToast color, header icon, content and header template, etc.

**Comment by crdo** [comment, marked as answer]
Hi @econner20, have a look at the discussion at https://github.com/havit/Havit.Blazor/issues/697
